src/braket/default_simulator/openqasm/parser/generated/braketPragmasParserVisitor.py

```diff
@@ -0,0 +1,508 @@
+# Generated from braketPragmasParser.g4 by ANTLR 4.9
+from antlr4 import *
+if __name__ is not None and "." in __name__:
+    from .braketPragmasParser import braketPragmasParser
+else:
+    from braketPragmasParser import braketPragmasParser
+
+# This class defines a complete generic visitor for a parse tree produced by braketPragmasParser.
+
+class braketPragmasParserVisitor(ParseTreeVisitor):
```

Comment:
Is it possible to generate this to `BraketPragmasParserVisitor` (and for the file names to be snake_case)?

Reply:
Yes we can and this bugs me too haha. I changed the casing to match openqasm's grammar file names. I think we should keep it consistent, but I could just as easily change qasm3 to Qasm3. Unfortunately I don't know of any way to change the generated file names

Reply:
Actually, I'm not convinced consistency between our grammar name and qasm3 matters. I'll just change ours to simplify future updates to the openqasm grammar version